frontend/src/components/Question.vue

```diff
@@ -93,7 +134,10 @@ const questionDifficultyColor: ComputedRef = computed(
             </navigation-button>
             <navigation-button
                 v-if="!props.questionsFlow.isLastQuestion"
-                @click="goToNextQuestion"
+                @click="
+                    resetHint();
```

Comment:
Is this really necessary? If I've already displayed the hint, whether I go back or forth doesn't matter

Reply:
Well seen... And as the user can hide again the hint himself... Okay let me change this.